ext/pdo/tests/bug_78227.phpt

```diff
@@ -0,0 +1,27 @@
+--TEST--
```

Comment:
Since this is a general PDO test case, it need to be adapted to work with all drivers (which support RENAME COLUMN statements). See, for instance, [pdo_001.phpt](https://github.com/php/php-src/blob/master/ext/pdo/tests/pdo_001.phpt) on how to accomplish this.